documentation/developers/getting-started.md

```diff
@@ -11,15 +12,16 @@ The repository is a monorepo which contains multiple components.
 It has been structured into separate components to streamline collaboration, whilst providing autonomy and tailored workflows without the overhead of multiple repositories. Each component contains a README with specific getting started instructions.
 
 ## Build & deployment 
+[comment]: <> (Should we hyperlink the text to the DevOps board?)
+Continuous integration, delivery and testing is automated via Azure Pipelines. Terraform is being used as the tool of choice for Infrastructure as Code (IaC) to allows for the build, change, and versioning of the infrastructure safely and efficiently.
 
-Continuous integration, delivery and testing is automated via Azure Pipelines. Terraform is being used as the tool of choice for infrastructure as code (IaC) to allows for the build, change, and versioning of the infrastructure safely and efficiently.
-
-Each component in the repository contains 'pipelines' and 'terraform' (where needed) folders.
+Each component in the repository contains 'pipelines' and 'terraform' folders where needed.
 
 ### Quality checks
 
 The following quality checks are automated in the pipelines, occurring in the build stage on any run (both PR and merge);
-
+[comment]: <> (Why is our testing coverage so low? Shouldn't we be aiming for the high 90% mark?)
```

Comment:
Should we amend the text to list our targeted coverage then?

Reply:
75% is a good target. I'd sooner we as a team focus on quality tests rather than a specific coverage target.  Code coverage can be faked.
 
 I think we should update the documentation when we update the pipeline, so that we keep things aligned.

Reply:
Forgetting about documentation is a general problem - we just need to stay on top of it.

Code coverage isn't really a measure of quality. Anything less than 100% could be considered low because you want to be sure everything is tested - however this risks introducing bad behaviours and completely devalues the pipeline check.

Reply:
If it is a general problem then why not remedy it now than introduce the risk of missing it later down the line? 

Agreed, but wouldn't we be better aiming for lots of quality checks rather than a few?

Reply:
Adding a target doesn't remedy the documentation issue. Documentation should be accurate,

Changing the percentage doesn't add more quality checks.